tracim/tracim
# Feat/4680  add link to contents

#4680 

To do after backend integration:
- Update the `getMyselfKnownContent` function on `frontend_lib/src/action.async.js`
  - Be careful, an empty string should return all contents (limites by `limit`)
- Manually test
- Check/update cypress tests
  - as the current version uses advanced search, which is not available on Travis, the tests fail in ci, if it continues to fail after the backend, please fix `functionnal_tests/cypress/integration/links/...`
  - pay a little more attention to the `autocomplete_advanced_edition_spec.js` test, because in local the `waitForTinyMCELoaded` function was not working, so it was developed without this function
  
## Checkpoints



**For developers**

  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified

**For code reviewers**


**For testers**

- [x] Manual, quality tests WILL BE DONE during the sprint integration
